### PR TITLE
Added Address parameter

### LINKED ISSUE #4: Add support for Address parameter
Thanks for the great plugin! I'd really like to be able to specify the `Address` parameter, as I am running uwsgi in separate containers than my consul agent and thus different IP Addresses. Without specifying an address, Consul assumes that [the address is of the agent](https://www.consul.io/docs/agent/http/agent.html#agent_service_register). Any chance that could be added?

### PR DESCRIPTION
**Pull request by micahhausler**
Closes #4

### CODE REVIEW

**Comment by unbit** [comment]
@ultrabug it looks good to me, care to merge ?

File: consul.c [hunk 1]
```diff
@@ -209,6 +210,7 @@ static void consul_setup() {
 			"id", &ucs->id,
 			"name", &ucs->name,
 			"port", &ucs->port,
+			"address", &ucs->address,
```
Comment (ultrabug) [inline consul.c]: Just for QA (unless it has a bad meaning) could you please order the parameters alphabetically and by so move this above the "id" ?

**Comment by ultrabug** [comment]
Quick QA comment if possible plz ? thanks @micahhausler

**Comment by micahhausler** [comment]
@ultrabug @unbit Should be good to go

File: README.md [hunk 2]
```diff
@@ -32,6 +32,7 @@ Optional key options:
 * `debug` : print http transactions in logs, for debugging
 * `deregister_url` : the api url for deregistering services, if not specified is built as url+/v1/agent/service/deregister/+id
 * `id` : the service instance unique id, used if multiple services share the same name
+* `address` : the service instance address. Defaults to the consul agent address if not specified
```
Comment (ultrabug) [inline README.md]: forgot to sort here please ;)

**Comment by micahhausler** [comment]
Done!

File: README.md [hunk 3]
```diff
@@ -31,6 +31,7 @@ Optional key options:
 * `check_url` : the api url for registering the service healthcheck, if not specified is built as url+/v1/agent/check/pass/service:+id or name
 * `debug` : print http transactions in logs, for debugging
 * `deregister_url` : the api url for deregistering services, if not specified is built as url+/v1/agent/service/deregister/+id
+* `address` : the service instance address. Defaults to the consul agent address if not specified
```
Comment (ultrabug) [inline README.md]: sorry, I meant on the whole list here

Comment (micahhausler) [inline README.md]: Oh ok! Sort all in just the README, or the struct too?

Comment (ultrabug) [inline README.md]: I'm no C expert but if it can be done on the struct too, I guess it's better ?

**Comment by micahhausler** [comment]
@ultrabug everything compiled/ran correctly for me as-is

**Comment by ultrabug** [comment]
Awesome work @micahhausler thank you very much !

**Pull request merged**